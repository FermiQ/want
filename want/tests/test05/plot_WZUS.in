
 &INPUT
   prefix = 'wurtzite_US'
   postfix = '_WanT'
   output_fmt = 'xsf'
   datatype = 'modulus'
   collect_wf = .TRUE.
   wann = '9-12'
   r1min = -1.0
   r1max =  1.0
   r2min = -1.0
   r2max =  1.0
 /

