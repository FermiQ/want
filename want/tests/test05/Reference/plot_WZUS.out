  ======================================================================
              =                                            =
              =     *** WanT *** Wannier Transport Code    =
              =        (www.wannier-transport.org)         =
              =      Ultra Soft Pseudopotential Implem.    =
              =                                            =
  ======================================================================


  Program <plot>  v. 2.0.0  starts ...
  Date 17Nov2005 at  1:12:54 

  Arch. / Op.Sys: LINUX
        Compiler: INTEL


  PW-DFT data read from file: wurtzite_US.export/index.xml
  Subspace data read from file: wurtzite_US_WanT.space
  Wannier data read from file: wurtzite_US_WanT.wan


  ======================================================================
  =                              DFT data                              =
  ======================================================================

<LATTICE>
  Alat  =       5.8748000 (Bohr)
  Alat  =       3.1088090 (Ang )
  Omega =     856.2094872 (Bohr^3)
  Omega =     126.8770041 (Ang^3 )

  Crystal axes:
                in units of Bohr                 in Alat units
    a(1) = (  5.8748  0.0000  0.0000 )     (   1.0000  0.0000  0.0000 )
    a(2) = ( -2.9374  5.0877  0.0000 )     (  -0.5000  0.8660  0.0000 )
    a(3) = (  0.0000  0.0000 28.6460 )     (   0.0000  0.0000  4.8761 )
  Crystal axes: (Ang)
    a(1) = (  3.1088  0.0000  0.0000 )
    a(2) = ( -1.5544  2.6923  0.0000 )
    a(3) = (  0.0000  0.0000 15.1588 )

   Reciprocal lattice vectors:
                in units of Bohr^-1              in 2Pi/Alat units
    b(1) = (  1.0695  0.6175  0.0000 )     (   1.0000  0.5774  0.0000 )
    b(2) = (  0.0000  1.2350  0.0000 )     (   0.0000  1.1547  0.0000 )
    b(3) = (  0.0000  0.0000  0.2193 )     (   0.0000  0.0000  0.2051 )
</LATTICE>

<K-POINTS>
  nkpts =   32
  Monkhorst-Pack grid:      nk = (  4  4  2 ),      shift = (  0  0  0 )
  K-point calculation: (cart. coord. in Bohr^-1)
    k point   1:   (   0.00000  0.00000  0.00000 ),   weight =   0.0312
    k point   2:   (   0.00000  0.30874  0.00000 ),   weight =   0.0312
    k point   3:   (   0.00000 -0.30874  0.00000 ),   weight =   0.0312
    k point   4:   (   0.00000 -0.61748  0.00000 ),   weight =   0.0312
    k point   5:   (   0.26738  0.15437  0.00000 ),   weight =   0.0312
    k point   6:   (   0.26738  0.46311  0.00000 ),   weight =   0.0312
    k point   7:   (   0.26738 -0.46311  0.00000 ),   weight =   0.0312
    k point   8:   (   0.26738 -0.15437  0.00000 ),   weight =   0.0312
    k point   9:   (  -0.26738 -0.15437  0.00000 ),   weight =   0.0312
    k point  10:   (  -0.26738  0.15437  0.00000 ),   weight =   0.0312
    k point  11:   (  -0.26738 -0.77186  0.00000 ),   weight =   0.0312
    k point  12:   (  -0.26738 -0.46311  0.00000 ),   weight =   0.0312
    k point  13:   (  -0.53476 -0.30874  0.00000 ),   weight =   0.0312
    k point  14:   (  -0.53476  0.00000  0.00000 ),   weight =   0.0312
    k point  15:   (  -0.53476 -0.92623  0.00000 ),   weight =   0.0312
    k point  16:   (  -0.53476 -0.61748  0.00000 ),   weight =   0.0312
    k point  17:   (   0.00000  0.00000  0.10967 ),   weight =   0.0312
    k point  18:   (   0.00000  0.30874  0.10967 ),   weight =   0.0312
    k point  19:   (   0.00000 -0.30874  0.10967 ),   weight =   0.0312
    k point  20:   (   0.00000 -0.61748  0.10967 ),   weight =   0.0312
    k point  21:   (   0.26738  0.15437  0.10967 ),   weight =   0.0312
    k point  22:   (   0.26738  0.46311  0.10967 ),   weight =   0.0312
    k point  23:   (   0.26738 -0.46311  0.10967 ),   weight =   0.0312
    k point  24:   (   0.26738 -0.15437  0.10967 ),   weight =   0.0312
    k point  25:   (  -0.26738 -0.15437  0.10967 ),   weight =   0.0312
    k point  26:   (  -0.26738  0.15437  0.10967 ),   weight =   0.0312
    k point  27:   (  -0.26738 -0.77186  0.10967 ),   weight =   0.0312
    k point  28:   (  -0.26738 -0.46311  0.10967 ),   weight =   0.0312
    k point  29:   (  -0.53476 -0.30874  0.10967 ),   weight =   0.0312
    k point  30:   (  -0.53476  0.00000  0.10967 ),   weight =   0.0312
    k point  31:   (  -0.53476 -0.92623  0.10967 ),   weight =   0.0312
    k point  32:   (  -0.53476 -0.61748  0.10967 ),   weight =   0.0312
</K-POINTS>


  WARNING: USPP not fully implemented


  Reading density G-grid from file: wurtzite_US.export/index.xml
  Reading Wfc grids from file: wurtzite_US.export/index.xml


          Energy cut-off for wfcs =         25.00 (Ry)
                         for rho  =        200.00 (Ry)
      Total number of PW for rho  =      40923
        Max number of PW for wfc  =       1834
      Total number of PW for wfcs =       3059
        FFT grid components (rho) =  (    27   27  135 )


  ======================================================================
  =                     Wannier function plotting                      =
  ======================================================================


  nplot =    4 Wannier func.s to be plotted
     wf (   1 ) =    9
     wf (   2 ) =   10
     wf (   3 ) =   11
     wf (   4 ) =   12

  Data type  :   modulus
  Output fmt :   xsf

  Plotting Cell dimensions [cryst. coord]:
       r1 :   -0.5000 -->   0.5000
       r2 :   -0.5000 -->   0.5000
       r3 :   -0.5000 -->   0.5000

  Grid dimensions:
      nrx :       -14 -->       14
      nry :       -14 -->       14
      nrz :       -68 -->       68


  Centers for the required Wannier functions:

        in cartesian coord (Bohr)
    Wf(   9 ) = (     0.000000    -0.000374    12.281704 )
    Wf(  10 ) = (     2.244191    -1.295932     8.651471 )
    Wf(  11 ) = (    -2.244198    -1.295932     8.651469 )
    Wf(  12 ) = (    -0.000003     2.592196     8.645838 )
        in crystal coord
    Wf(   9 ) = (    -0.000037    -0.000073     0.428741 )
    Wf(  10 ) = (     0.254644    -0.254717     0.302014 )
    Wf(  11 ) = (    -0.509363    -0.254717     0.302014 )
    Wf(  12 ) = (     0.254749     0.509500     0.301817 )

  Locating WFs: 
  Plotting Cell dimensions [cryst. coord]:
       r1 :   -0.5000 -->   0.5000
       r2 :   -0.5000 -->   0.5000
       r3 :   -0.5000 -->   0.5000

  New center positions [cryst. coord]:
        in crystal coord
    Wf(   9 ) = (    -0.000037    -0.000073     0.428741 )
    Wf(  10 ) = (     0.254644    -0.254717     0.302014 )
    Wf(  11 ) = (     0.490637    -0.254717     0.302014 )
    Wf(  12 ) = (     0.254749    -0.490500     0.301817 )


  Atoms in the selected cell: (cart. coord. in Bohr)
     Al   tau(   1 ) = (   0.0000000   0.0000000   0.0000000 )
     Al   tau(   2 ) = (   2.9373971  -1.6959070   4.7743346 )
     N    tau(   3 ) = (   0.0000000   0.0000000   3.5807438 )
     N    tau(   4 ) = (   2.9373971  -1.6959070   8.3550783 )
     Al   tau(   5 ) = (   0.0000000   0.0000000   9.5486405 )
     Al   tau(   6 ) = (   2.9373971  -1.6959070 -14.3229751 )
     Al   tau(   7 ) = (   2.9373971  -1.6959070  14.3229751 )
     N    tau(   8 ) = (   0.0000000   0.0000000  13.1293843 )
     N    tau(   9 ) = (   2.9373971  -1.6959070 -10.7422313 )
     Al   tau(  10 ) = (   0.0000000   0.0000000  -9.5486405 )
     Al   tau(  11 ) = (   2.9373971  -1.6959070  -4.7743346 )
     N    tau(  12 ) = (   0.0000000   0.0000000  -5.9678967 )
     N    tau(  13 ) = (   2.9373971  -1.6959070  -1.1935908 )


    Wfc Fourier Transf. for k-point    1
                              Total time spent up to now :    23.20 secs

    Wfc Fourier Transf. for k-point    2
                              Total time spent up to now :    30.39 secs

    Wfc Fourier Transf. for k-point    3
                              Total time spent up to now :    38.66 secs

    Wfc Fourier Transf. for k-point    4
                              Total time spent up to now :    46.90 secs

    Wfc Fourier Transf. for k-point    5
                              Total time spent up to now :    54.91 secs

    Wfc Fourier Transf. for k-point    6
                              Total time spent up to now :    63.59 secs

    Wfc Fourier Transf. for k-point    7
                              Total time spent up to now :    71.82 secs

    Wfc Fourier Transf. for k-point    8
                              Total time spent up to now :    80.52 secs

    Wfc Fourier Transf. for k-point    9
                              Total time spent up to now :    85.73 secs

    Wfc Fourier Transf. for k-point   10
                              Total time spent up to now :    94.08 secs

    Wfc Fourier Transf. for k-point   11
                              Total time spent up to now :   102.33 secs

    Wfc Fourier Transf. for k-point   12
                              Total time spent up to now :   110.53 secs

    Wfc Fourier Transf. for k-point   13
                              Total time spent up to now :   115.70 secs

    Wfc Fourier Transf. for k-point   14
                              Total time spent up to now :   124.62 secs

    Wfc Fourier Transf. for k-point   15
                              Total time spent up to now :   132.75 secs

    Wfc Fourier Transf. for k-point   16
                              Total time spent up to now :   141.34 secs

    Wfc Fourier Transf. for k-point   17
                              Total time spent up to now :   149.61 secs

    Wfc Fourier Transf. for k-point   18
                              Total time spent up to now :   154.86 secs

    Wfc Fourier Transf. for k-point   19
                              Total time spent up to now :   162.97 secs

    Wfc Fourier Transf. for k-point   20
                              Total time spent up to now :   168.34 secs

    Wfc Fourier Transf. for k-point   21
                              Total time spent up to now :   176.30 secs

    Wfc Fourier Transf. for k-point   22
                              Total time spent up to now :   181.67 secs

    Wfc Fourier Transf. for k-point   23
                              Total time spent up to now :   190.26 secs

    Wfc Fourier Transf. for k-point   24
                              Total time spent up to now :   198.30 secs

    Wfc Fourier Transf. for k-point   25
                              Total time spent up to now :   206.94 secs

    Wfc Fourier Transf. for k-point   26
                              Total time spent up to now :   215.10 secs

    Wfc Fourier Transf. for k-point   27
                              Total time spent up to now :   220.23 secs

    Wfc Fourier Transf. for k-point   28
                              Total time spent up to now :   228.41 secs

    Wfc Fourier Transf. for k-point   29
                              Total time spent up to now :   235.48 secs

    Wfc Fourier Transf. for k-point   30
                              Total time spent up to now :   244.16 secs

    Wfc Fourier Transf. for k-point   31
                              Total time spent up to now :   253.53 secs

    Wfc Fourier Transf. for k-point   32
                              Total time spent up to now :   259.15 secs


  writing WF(   9) plot on file: .//wurtzite_US_WanT_WFM009.xsf
  writing WF(  10) plot on file: .//wurtzite_US_WanT_WFM010.xsf
  writing WF(  11) plot on file: .//wurtzite_US_WanT_WFM011.xsf
  writing WF(  12) plot on file: .//wurtzite_US_WanT_WFM012.xsf

  ======================================================================

   <global routines>
             clock number :     7

                     plot :  4m24.00s CPU 
 
                want_init :     0.16s CPU
          ggrids_read_ext :     0.15s CPU
           wfc_data_kread :     1.38s CPU (      32 calls,   0.043 s avg)
               kwann_calc :   221.90s CPU (     128 calls,   1.734 s avg)
                   cfft3d :    14.74s CPU (     128 calls,   0.115 s avg)
               cwann_calc :     6.42s CPU (     128 calls,   0.050 s avg)


